media/sounds/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_Critical_Stop.wav DESTINATION reactos/media FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_LogOn.wav DESTINATION reactos/media FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/ReactOS_Notify.wav DESTINATION reactos/media FOR all)
```

Comment:
Should the new files not be registered in `boot/bootdata/hivedef.inf` ?

Reply:
> Please revert this and other registry entries. We don't always need to set them. They are automatically set at 2nd setup stage for Workstation configuration if the appropriate sound file exists. But for Server, we keep the default behaviour, without sounds enabled, excepting logon sound. Same as in Windows Server 2003 SP2.

From PR [#4209](https://github.com/reactos/reactos/pull/4209)

Reply:
On non-server versions of Windows, the sounds are registered in "Windows Default" profile.

Would it be better if we have "Windows Server 2003 Default" profile separated from "ReactOS Default" profile?

Reply:
Just FYI, Workstation vs. Server profile is being selected at the 2nd stage setup.